network/sites.csv

```diff
@@ -1752,6 +1752,7 @@ RL006,MC02,-29.264281,-177.921582,10,,WGS84,External GPS Device,1980-08-01T00:00
 RL006,MC03,-29.264281,-177.921582,10,,WGS84,External GPS Device,1980-08-01T00:00:00Z,2006-03-16T00:00:00Z
 RL006,MC04,-29.264281,-177.921582,10,,WGS84,External GPS Device,1980-08-01T00:00:00Z,2006-03-16T00:00:00Z
 RL006,MC05,-29.264281,-177.921582,10,,WGS84,External GPS Device,1980-08-01T00:00:00Z,2006-03-16T00:00:00Z
+RLGLK,04,-29.26068,-177.918038,30,,WGS84,External GPS Device,2023-01-30T02:59:59Z,9999-01-01T00:00:00Z
```

Comment:
@rumachan , for the coordinates:
the undeployed water and temperature sensors at the sites have these as coordinates:
```
GLKZ,40,-29.26247,-177.91978,10,,WGS84,External GPS Device,2009-05-19T00:00:00Z,2023-01-30T20:17:32Z
GLKZ,80,-29.2624,-177.91978,10,,WGS84,External GPS Device,2009-05-02T00:00:00Z,2023-01-30T20:17:32Z
GLKZ,81,-29.26231,-177.92025,10,,WGS84,External GPS Device,2009-05-02T00:00:00Z,2023-01-30T20:17:32Z
```

do you know if RLGLK sensors have been placed in a different position now? (same comment valid for the coordinates in stations.csv)